### ISSUE TITLE
Can I use UICollectionViewDelegate & MSPeekImplementationDelegate together?

### ISSUE THREAD

**Issue by JhonnyTawk** [MaherKSantina/MSPeekCollectionViewDelegateImplementation#30]
I am not able to use UICollectionViewDelegate when I am using MSPeekImplementationDelegate

**Comment by MaherKSantina** [comment]
Hello Jhonny, 
Thank you for raising this issue.
So the `MSPeekCollectionViewDelegateImplementation` is actually an implementation for the `UICollectionViewDelegate`. I'm understanding you want to use the implementation but also still implement the delegate methods found in the `UICollectionViewDelegate` is that correct?
If this is the case, you can actually do that by subclassing the `MSPeekCollectionViewDelegateImplementation` and adding your own overridden methods. Please check [Subclassing](https://github.com/MaherKSantina/MSPeekCollectionViewDelegateImplementation#subclassing) section for more information.

**Comment by JhonnyTawk** [comment]
I am getting this issue 
<img width="1330" alt="screen shot 2019-01-10 at 1 12 15 pm" src="https://user-images.githubusercontent.com/38781782/50965005-7121a580-14d9-11e9-968d-60ffdd721c97.png">

**Comment by JhonnyTawk** [comment]
I am only inheriting UICollectionViewDataSource and now MSPeekCollectionViewDelegateImplementation

**Comment by JhonnyTawk** [comment]
Thank you!
It worked fine after that implementation. The main issue was I want to implement willDisplayCell but when I inherited UICollectionViewDelegate

**Comment by MaherKSantina** [comment]
Yes you're right it wouldn't work this way. I'm thinking of a way to make it easier to do that. Will update the repo if I find anything 👍
